package.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "@openinf/.github",
+  "version": "0.0.0-development",
+  "description": "",
+  "main": "",
+  "directories": {
+    "doc": "doc",
+    "lib": "lib",
+    "test": "test"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

```suggestion
    "test": "exit 0"
```

Not necessary in this change, but have (have not purged in a while, but put much thought into a testing this package yet yet, but that would be next&hellip;